openrr-teleop/src/ik.rs

```diff
@@ -226,3 +226,39 @@ fn default_move_step_linear() -> [f64; 3] {
 fn default_step_duration_secs() -> f64 {
     0.1
 }
+
+#[cfg(test)]
+mod tests {
+    use assert_approx_eq::*;
+
+    use super::*;
+
+    #[test]
+    fn test_default_move_step_angular() {
+        let def = default_move_step_angular();
+        let mut def_iter = def.iter();
+
+        assert_approx_eq!(def_iter.next().unwrap(), 0.05_f64);
+        assert_approx_eq!(def_iter.next().unwrap(), 0.05_f64);
+        assert_approx_eq!(def_iter.next().unwrap(), 0.17_f64);
+        assert_eq!(def_iter.next(), None);
```

Comment:
How about using elements of the array directory (e.g. `def[0]`) instead of getting them via iterator?
I think it would become simpler and be easier to convey your intentions.

Reply:
I'm sorry, you are right very much.
I do so because I felt like it. And no reason.

I'm going to redo at next time.